composer.json

```diff
@@ -23,11 +23,11 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.3",
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
         "doctrine/orm": "^2.5",
         "doctrine/persistence": "^1.3.4",
-        "sonata-project/admin-bundle": "^3.71",
+        "sonata-project/admin-bundle": "dev-master",
```

Comment:
```suggestion
        "sonata-project/admin-bundle": "4.x-dev",
```

Reply:
4.0@dev isnt better?

Reply:
4.0@dev doesn't exist, I'll use `4.x-dev`.

Reply:
> 4.0@dev doesn't exist, I'll use `4.x-dev`.

This pull request mean that release sonata admin version 4 is coming soon ?

Reply:
> > 4.0@dev doesn't exist, I'll use `4.x-dev`.
> 
> This pull request mean that release sonata admin version 4 is coming soon ?

Soon is a big word when the last major release was made years ago.
But I would like to try to release the 4.0 before the end of year.

This PR will allow to use sonataAdmin@master and sonataDoctrineORMAdmin@master together, this way we'll be able to know if something doesn't work on SonataAdmin@master.

Reply:
> > > 4.0@dev doesn't exist, I'll use `4.x-dev`.
> > 
> > This pull request mean that release sonata admin version 4 is coming soon ?
> 
> Soon is a big word when the last major release was made years ago.
> But I would like to try to release the 4.0 before the end of year.
> 
> This PR will allow to use sonataAdmin@master and sonataDoctrineORMAdmin@master together, this way we'll be able to know if something doesn't work on SonataAdmin@master. 

Okay thanks for your explanation,

I understand this release is not easy and have lots of thinks to do and test

Do you think is too early to use version master (4.x-dev) of sonataAdmin and sonataDoctrineORMAdmin together or is not a really stable version for the moment ?

I just use this 2 bundles and sonataMediaAdmin

Reply:
I would say yes and no.

It's too early because we have some work to do on master branch, you'll have multiple BC break, and since the branch was never tested in real condition I can't say how good it's work.

But personally I'm gonna try to use it in order to find some bug and to know what we need to do in order to have a stable 4.0 version.

So if you want to try to use the master version, it will be a good help for us.